### ISSUE TITLE
Lots of errors after upgrading from v. 1.18.1

### ISSUE THREAD

**Issue by yekver** [scrumpy/tiptap#327]
After upgrading from 1.18.1 -> 1.20.1 I see lots of errors in browser console:
- `Error in data(): "RangeError: Adding different instances of a keyed plugin (plugin$)"`
- `TypeError: Cannot read property 'getHTML' of undefined`
- `Property or method "editor" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.`

much more errors, but all of them are about the undefined instance property. 

Errors are gone only after I delete from `data` this line: `editor: new Editor()`. I tried to remove my node_modules directory and reinstall all modules from scratch, but no luck.

Also It's not clear how and why Vue lost access to all component properties defined in `data`.

**Comment by philippkuehn** [comment]
Duplicate of #305
